android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -290,49 +291,73 @@ private void setMACFiltering(MethodCall poCall, Result poResult) {
      * (e.g., {@code 01a243f405}).
      */
     private void getWiFiAPSSID(Result poResult) {
-        android.net.wifi.WifiConfiguration oWiFiConfig = moWiFiAPManager.getWifiApConfiguration();
-        if (oWiFiConfig != null && oWiFiConfig.SSID != null) {
-            poResult.success(oWiFiConfig.SSID);
-            return;
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            android.net.wifi.WifiConfiguration oWiFiConfig = moWiFiAPManager.getWifiApConfiguration();
+
+            if (oWiFiConfig != null && oWiFiConfig.SSID != null) {
+                poResult.success(oWiFiConfig.SSID);
+                return;
+            }
+
+            poResult.error("Exception", "SSID not found", null);
         }
-        poResult.error("Exception", "SSID not found", null);
+            else {
+                poResult.error("Exception [getWiFiAPSSID]", "Getting SSID name is not supported on API level >= 26", null);
+            }
```

Comment:
can we not return "random" SSID generated by LocalOnlyHostspot?

Reply:
I believe it should be possible. `onStarted` returns a `LocalOnlyHotspotReservation` on which with `getSoftApConfiguration` you should get a `SoftApConfiguration` which then has `getSsid()` (and `getPassphrase()` for the comment below). 

Of course this works only if the `LocalOnlyHotspot` has been activated (since as usual we are just "requesting") so a bit of different cases handling needs to be done I believe.

Reply:
will be tracked this with #134